PassTime.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+include "classes.php";
+
+include "commonUI.php";
+
+
+$time = $_GET["restDays"];
+
+for ($i=0; $i < $time; $i++) { 
+
+    $_SESSION["playerJourney"]->incrementDay();
+
+}
+
+$dateString = $_SESSION["playerJourney"]->_month . " " . $_SESSION["playerJourney"]->_date;
```

Comment:
Output the date in the format 1848-MONTH-DAY so it's easy to make a JS object on the frontend